x-pack/test/functional/page_objects/gis_page.ts

```diff
@@ -296,10 +296,28 @@ export class GisPageObject extends FtrService {
     };
   }
 
+  // This method is also used by upgrade testing which is not part of PR testing
+  // Please keep in mind when udpating, removing or adding to this method
+  // upgrade needs to be tested too
+  async clearLegendTooltip() {
+    const isTooltipOpen = await this.testSubjects.exists(`layerTocTooltip`, { timeout: 5000 });
```

Comment:
One more thing, change this exists timeout be shorter? how about 1000 or 1500?

Reply:
What I noticed when testing against cloud is things in the UI are slower, take longer to show up, so if it is too short we may miss it and it will cause it to be flaky